mkosi/config.py

```diff
@@ -1677,6 +1678,18 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         parse=config_make_list_parser(delimiter=" ", unescape=True),
         help="Set an environment variable when running scripts",
     ),
+    MkosiConfigSetting(
+        dest="environment_files",
+        long="--environment-files",
```

Comment:
Please make this `--env-file` i.e. we use singular instead of plural for CLI options and something make them a bit shorter